sdk/nodejs/api/test/api.spec.ts

```diff
@@ -233,4 +233,25 @@ describe("NodeJS SDK api", function () {
       `{ container { from (address: "alpine") { withWorkdir (path: "/foo") { withMountedDirectory (path: "/foo",source: {"_queryTree":[{operation:"host"},{operation:"directory",args:{path:"foo"}}],clientHost:"127.0.0.1:8080",sessionToken:"",client:{url:"http://127.0.0.1:8080/query",options:{headers:{Authorization:"Basic Og=="}}}}) { withMountedDirectory (path: "/bar",source: {"_queryTree":[{operation:"host"},{operation:"directory",args:{path:"bar"}}],clientHost:"127.0.0.1:8080",sessionToken:"",client:{url:"http://127.0.0.1:8080/query",options:{headers:{Authorization:"Basic Og=="}}}}) { withExec (args: ["blah"]) }}}}} }`
     )
   })
+
+  it("Compute nested arguments", async function () {
+    this.timeout(60000)
+
+    await connect(async (client) => {
+      const node = client.directory().withNewFile(
+        "Dockerfile",
+        `
+            FROM node:alpine
+            RUN node --version
+          `
+      )
+
+      const code = await client
+        .container()
+        .build(node, { buildArgs: [{ value: "foo", name: "test" }] })
+        .exitCode()
+
+      assert.strictEqual(code, null)
```

Comment:
I think the assert is unnecessary since if this doesn't fail, the await just throws an error.

Reply:
If I'm not wrong every tests in this file has an assert. Even if our first goal is not to check the exitCode(), I would prefer to stay consistent.